diamond_agent/tests/resources/blueprint/multi_node.yaml

```diff
@@ -25,7 +25,6 @@ node_templates:
                         diamond_config: { get_input: diamond_config }
                 start: diamond.diamond_agent.tasks.start
                 stop: diamond.diamond_agent.tasks.stop
-                uninstall: diamond.diamond_agent.tasks.uninstall
```

Comment:
Is there a way to call a single operation from the test? I've only seen workflows executions. The check would have to be between del_collectors and uninstall operations.